cg/apps/demultiplex/sample_sheet/sample_sheet_creator.py

```diff
@@ -38,7 +44,7 @@ def __init__(
 
     @property
     def valid_indexes(self) -> List[Index]:
-        return index.get_valid_indexes(dual_indexes_only=True)
+        return get_valid_indexes(dual_indexes_only=True)
 
     def add_dummy_samples(self) -> None:
```

Comment:
Is this a helper method for tests?

Reply:
No, previous flow cells needed all indexes to be used so for the unused ones we add dummy samples